mypyc/lib-rt/misc_ops.c

```diff
@@ -635,6 +636,22 @@ int CPyStatics_Initialize(PyObject **statics,
             *result++ = obj;
         }
     }
+    if (frozensets) {
+        int num = *frozensets++;
+        while (num-- > 0) {
+            int num_items = *frozensets++;
+            PyObject *obj = PyFrozenSet_New(NULL);
+            if (obj == NULL) {
+                return -1;
+            }
+            for (int i = 0; i < num_items; i++) {
+                PyObject *item = statics[*frozensets++];
+                Py_INCREF(item);
+                PySet_Add(obj, item);
```

Comment:
Check the return value of `PySet_Add`.

Reply:
Whoops I totally forgot about this. Good catch! Addressed in https://github.com/python/mypy/pull/14409/commits/9cdfc98b25058f0bd4c8e45c44aeea97dd46a04e.